yarn.lock

```diff
@@ -11752,13 +11740,16 @@ walker@^1.0.7, walker@~1.0.5:
   dependencies:
     makeerror "1.0.x"
 
+<<<<<<< HEAD
```

Comment:
Uh, oh, this shouldn't be in here! Looks like you've got some artifacts from a merge conflict - probably just regenerate this file...